test/util/util.go

```diff
@@ -297,12 +306,25 @@ func AdmitWorkload(ctx context.Context, k8sClient client.Client, wl *kueue.Workl
 		return fmt.Errorf("admission can't be null")
 	}
 	wl.Status.Admission = admission
-	wl.Status.Conditions = []metav1.Condition{{
+	apimeta.SetStatusCondition(&wl.Status.Conditions, metav1.Condition{
```

Comment:
Oops, it looks like we have 3 functions that do roughly the same. Let's only keep one: `SetAdmission` and make it work for Admit and Evict.